bin/import_fisher.py

```diff
@@ -236,14 +237,18 @@ def _split_and_resample_wav(origAudio, start_time, stop_time, new_wav_file):
 
 
 def _split_sets(filelist):
-    # We initially split the entire set into 80% train and 20% test, then
-    # split the train set into 80% train and 20% validation.
+    """
```

Comment:
I'm wondering if this should not be shared and re-used from https://github.com/mozilla/DeepSpeech/blob/385c8c769bc4aed5e6979a239591486c44f3471d/training/deepspeech_training/util/importers.py